plugins/chat/db/migrate/20230201012734_create_chat_threading_models.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class CreateChatThreadingModels < ActiveRecord::Migration[7.0]
+  def change
+    create_table :chat_threads do |t|
+      t.integer :channel_id, null: false
+      t.integer :original_message_id, null: false
+      t.integer :original_message_user_id, null: false
+      t.integer :status, null: false, default: 0
+      t.string :title, null: true
```

Comment:
I really like that you put `null: true` here to indicate this is an explicit decision. 👍

Reply:
Thanks, I usually do this because I don't like that you have to know what AR is doing by default otherwise.